openlibrary/templates/account/create.html

```diff
@@ -43,36 +43,41 @@ <h1>$_("Sign Up")</h1>
         $if form.note:
             <div class="note">$form.note</div>
 
-        $def screenname_url(): $_('Your URL'): https://openlibrary.org/people/<span id="userUrl">$(form.username.value or _('screenname'))</span>
+        $def screenname_url(): $_('')
```

Comment:
```suggestion
```
This is no longer needed.  Please remove `suffix` from the `field` call on line 49, below.  Also, remove `suffix` from the `field` definition, and the `span` that displays `suffix`.